sd/install.ps1

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env pwsh
+
+##################
+# Install sd #
+##################
+
```

Comment:
This is missing `WEBI_SINGLE=true`... perhaps the example is missing it too. Oops.

Reply:
Indeed, it is missing from `_example/install.sh` also. But it's in all of the other packages that need it, such as `rg/install.sh`. You can see what it looks like there.

Reply:
WEBI_SINGLE=true is not in rg/install.sh https://github.com/webinstall/packages/blob/master/rg/install.sh